platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -937,6 +945,18 @@ public void setPaymentsManager(PaymentsManager mPaymentsManager) {
 	*/
 
 	// Audio
+	public boolean requestPermission(String p_name) {
+		if (p_name.equals("RECORD_AUDIO")) {
+			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Similar to above comment, will this check also be relevant for other types of permissions? If so, it should probably be done the other way around?
```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
	if (p_name.equals("RECORD_AUDIO")) {
```
or maybe even:
```
if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
	 // Not necessary, asked on install already
	return true;
}

if (p_name.equals("RECORD_AUDIO")) {
```

Reply:
Sound good, I'll change it